frontend/app/src/bounties/BountyDescription.tsx

```diff
@@ -148,6 +151,26 @@ const BountyDescription = (props: BountiesDescriptionProps) => {
               org_uuid={props.uuid}
             />
           </div>
+          {props.org_uuid && props.name && (
+            <Link to={`/org/bounties/${props.org_uuid}`} target="_blank">
+              <OrganizationWrap>
+                <img
```

Comment:
I see prop.img defined in two ways: 

img={props.image}
org_img={props.img}

I think the code used img instead of org_img here.

Reply:
@ecurrencyhodler I figured out how to fetch the org details and get the proper org image, now it shows up correctly. 

<img width="1201" alt="image" src="https://github.com/stakwork/sphinx-tribes/assets/43463/f7023fa1-f396-4ea6-8af8-209411d4cd2c">